src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-metadata-info/rgw-sync-metadata-info.component.scss

```
@@ -6,3 +6,7 @@ ul {
   flex-direction: column;
   list-style-type: none;
 }
+
+.toggleTipBtn {
```

why do we need this ? and also scss naming follow the following convention
```suggestion
.toggle-tip-btn {
```